src/tools/rust-analyzer/bench_data/glorious_old_parser

```diff
@@ -1,6 +1,6 @@
 use crate::ast::{AngleBracketedArgs, ParenthesizedArgs, AttrStyle, BareFnTy};
 use crate::ast::{GenericBound, TraitBoundModifier};
-use crate::ast::Unsafety;
```

Comment:
This is just a test file, not sure if it's worth updating it.

Reply:
It's not, no need to touch the r-a subtree here

Reply:
I mean, I don't think it's worth updating it even in the RA repo.

Reply:
Agreed, removed this change from the PR. Anyway, just at a glance seems that this test file isn't working as is in master, from the top of my head RA uses chalk's Safety enum, so importing Unsafety would just explode.